HW10/read.txt

```diff
@@ -0,0 +1,34 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
+
+
+ 
+ console.log (b);
+ var i=0;
+ y=[];
+ for(i=0;i<b.length;i++){
+    y[i]=(2/3*b[i]+1/2);
+    console.log (y[i]);
+    y[i]=y[i]+b[i];
```

Comment:
var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];

 console.log (b);
 var i=0;
 var y=[];
 var x=[];
 var z=0;
 var n=0;
 
 for(i=0;i<b.length;++i){
    y[z]=(2/3*b[i]+1/2);
    console.log (y[z]);
 }
 
for (i=0;i<b.length ;++i){
 	for (z=0;z<y.length;++z){
 	x[n]=(y[z]*b[i]);
 	console.log (x[n]);
 }
 
}